lib/tasks/buildThemes.js

```diff
@@ -66,7 +66,7 @@ async function buildThemeInWorker(options, taskUtil) {
 export default async function({
 	workspace, dependencies, taskUtil,
 	options: {
-		projectName, inputPattern, librariesPattern, themesPattern, compress, cssVariables, useWorkers = true
+		projectName, inputPattern, librariesPattern, themesPattern, compress, cssVariables, useWorkers = false
```

Comment:
add a `TODO for V4` to change default to true

Reply:
Done